docs/user-guide/user-defined-functions/reduce/reduce.md

```diff
@@ -66,7 +66,7 @@ chaining of windows, keyed streams, etc.
 ## Storage
 
 Reduce unlike map requires persistence. To support persistence user has to define the 
-`persistence` configuration. We replay the data stored in this store on pod startup if there has
+`persistence` configuration. We replay the data stored in this storage on pod startup if there has
```

Comment:
What I meant was, it's `storage` instead of `persistence`.  Maybe you are not talking about the property name, but it's confusing.